### PR TITLE
Share and protobuf updated  for arithmetic type

### PR DESCRIPTION
**Pull request by watason**
Share updated to generic type

types are  bool,int,float,double,other(string).

### CODE REVIEW

File: src/Proto/ComputationToComputationContainer/computation_to_computation.proto [hunk 1]
```diff
@@ -37,9 +44,16 @@ message Share {
...
+            int64 num64 = 5;
+            bytes byte = 6;
+            float f = 7;
+            double d = 8;
+            bool flag = 9;
+        }
         Address address_id = 2;
     }
-    repeated Share share_list = 1; // list of share
+    repeated Share share_list= 1;
```
Comment (come2ry) [inline src/Proto/ComputationToComputationContainer/computation_to_computation.proto:57]:
```suggestion
    repeated Share share_list = 1;
```

**Review by come2ry — CHANGES_REQUESTED** [review]

**Review by mdonaka — APPROVED** [review]

File: src/Proto/ComputationToComputationContainer/computation_to_computation.proto (hunk 1 above)
Comment (arukuka) [reply src/Proto/ComputationToComputationContainer/computation_to_computation.proto]: @come2ry Thank you for your review.  
It is fixed.  Please check these.

**Comment by come2ry** [comment]
LGTM

**Review by come2ry — APPROVED** [review]

**Comment by arukuka** [comment]
@watason Thank you for your contribution!  
Same as https://github.com/acompany-develop/QuickMPC/pull/23#pullrequestreview-1092723460, could you change the author/committer of commit if you don't mind?

> Thank you for your contribution! LGTM.
> 
> This is a quite minute thing, but there is a notational variation in the author of the commit.  
> Could you correct it if you don't mind?  
> Also, please register email address used to GitHub for showing icon. :bow:

**Comment by arukuka** [comment]
@watason  
Thank you for your quick response!  
I will merge it.

**Pull request merged**